build-wallet-electron.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+OS="`uname`"
+case $OS in
+  'Linux')
+   OS='Linux'
+   alias ls='ls --color=auto'
+    ;;
+  'FreeBSD')
+    OS='FreeBSD'
+    alias ls='ls -G'
+    ;;
+  'WindowsNT')
+    OS='Windows'
+    ;;
+  'Darwin') 
+    OS='Mac'
+    ;;
+  'arm64')
```

Comment:
No

Reply:
I think this needs to be addressed so i'm unresolving this conversation, `arm64` isn't an OS, its a chip instruction type so it doesn't make sense to match try and match this